### PR TITLE
Enable webnn wire for CI test

### PR DESCRIPTION
**Pull request by fujunwei**

### CODE REVIEW

**Comment by fujunwei** [comment]
@huningxin  PTAL, thanks.

**Review by huningxin — APPROVED** [review]
LGTM, @lisa0314 , PTAL, thanks.

File: src/webnn_wire/client/OperandArray.cpp [hunk 1]
```diff
@@ -20,14 +20,15 @@
 namespace webnn_wire::client {
 
     size_t OperandArray::Size() {
-        OperandArraySizeCmd cmd;
-        cmd.operandArrayId = this->id;
+        // OperandArraySizeCmd cmd;
```
Comment (lisa0314) [inline src/webnn_wire/client/OperandArray.cpp:23]: remove this line?

**Review by lisa0314 — APPROVED** [review]
LGTM. Thanks @fujunwei !

File: src/webnn_wire/client/OperandArray.cpp (hunk 1 above)
Comment (fujunwei) [reply src/webnn_wire/client/OperandArray.cpp]: Done, thanks for review.

**Comment by fujunwei** [comment]
Merge it.

**Pull request merged**